docs/run-without-docker.md

```diff
@@ -0,0 +1,60 @@
+# Run Node Locally without Docker
```

Comment:
```suggestion
# Running the Rollups Node locally
```
"without Docker" is misleading, since it is still necessary to build/run the dependencies

Reply:
Using Docker for the Node Services is "locally" also. "Nativelly" woul be a better term. Someone agrees?

Reply:
Natively makes sense.